### ISSUE TITLE
Can't find imports in LSP.plugin

### ISSUE THREAD

**Issue by ratijas** [sublimelsp/LSP-pyright#182]
Installed LSP-pyright, cloned this repo, opened directory in Sublime Text, saved project, edited project, set `"pyright.dev_environment": "sublime_text"`, opened file `LSP-pyright/plugin.py`. LSP can't find imports from ST packages like `from LSP.plugin import DottedDict`.

![image](https://user-images.githubusercontent.com/6737986/187083358-7ed06667-3785-4eda-9dca-a4e8f462025e.png)

I expected `pyright.dev_environment` option to take care of that. Is it possible to pass those modules from inside a package to an LSP without cloning the repo or manually unarchiving the package?

**Comment by jfcherng** [comment]
> I expected pyright.dev_environment option to take care of that. Is it possible to pass those modules from inside a package to an LSP without cloning the repo or manually unarchiving the package?

pyright can't take a zip.

**Comment by ratijas** [comment]
Huh, I was hoping that "res" in schemes would mean something like that.

**Comment by jfcherng** [comment]
res: is ST thing.

**Comment by ratijas** [comment]
So, an LSP plugin can't act like a proxy, fetching those things as needed?

**Comment by jfcherng** [comment]
yes, decompress them somewhere on your filesystem auto for you. no one implements.

**Comment by jfcherng** [comment]
Original discussion https://github.com/sublimelsp/LSP-pyright/issues/71

**Comment by rchl** [comment]
Would be good to figure out what is the .zip support in pyright about and how it's supposed to work. If it works then we could get away with just copying packages somewhere and renaming to `*.zip`. Would be a lot more efficient.
